### PR TITLE
Neka-Create new branch with navbar component

### LINKED ISSUE #35: Neka - NavBar Component [logo component, search bar component, navigation component {login & signup}] (SP: 5)
1. Code the Navbar with logo, search bar, navigation
2. Navigation will refer to Login and Signup Pages

searchbar-screenshot

### PR DESCRIPTION
**Pull request by istoga**
## What type of PR is this? (check all applicable)

- [ ] Feature
- [ ] Refactor
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [X] Other (please specify here: Create a component)
  
  
## Description of what this Pull Request does
1. Creation of NavBar component including logo, search bar and navigation links
2. Very basic styling of NavBar component
3. Addition of stylesheet for NavBar component
* logo was added as placeholder
* search bar was not made functional since teammates need to meet again to find out exactly what terms we will have available for users to search through, find out appropriate search constraints
* navigation links are not functional until I meet with teammates again to find out what pages/parts of the app we would like users/managers to be able to easy click through to

## Related Tickets & Documents
- Closes #35
- Related Issues: #40 
  
## QA Instructions, Screenshots, Recordings
(In this section, describe how to QA/review your PR. Feel free to use screenshots and/or short videos)
navbar-ui

  
## Added/updated tests?
- [ ] Yes
- [ ] Not Yet, and here is why: __explain here__
- [ ] I need help with tests
- [x] No, and here is why: Have not started doing any tests.

### CODE REVIEW

**Comment by TaylorDJones11** [comment]
Looks great @istoga ! The 'create listing' was in the manager's dashboard but this placement is much better in the navbar.

**Comment by istoga** [comment]
Thanks @TaylorDJones11 I wasn't sure which links to place in the navigation, but if that does look better there then cool!

File: src/components/Header/NavBar.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            <Image
+                src="/vercel.svg"
+                alt="Vercel Logo"
+                className={styles.vercelLogo}
+                width={100}
+                height={24}
+                priority
+              />
+            
+            <form>
```
Comment (jjleigh) [inline src/components/Header/NavBar.js:18]: I would move this into its own component and import it here.

Comment (istoga) [reply src/components/Header/NavBar.js:18]: I can definitely move the form into it's own component, thanks @jjleigh

**Comment by ibimina** [comment]
The Navbar component looks good, but it looks more like a  navbar component for a signed in manager

File: src/components/Header/NavBar.js (hunk 1 above)
Comment (msarit) [reply src/components/Header/NavBar.js:18]: Remember to create a ticket for this task!

**Comment by ibimina** [comment]
@istoga your branch is some commits behind main branch, Kindly update your branch

File: src/components/Header/NavBar.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import Link from 'next/link';
```
Comment (msarit) [inline src/components/Header/NavBar.js:1]: I am seeing two different import approaches:
- `next/<item>`
- `@/path/to/item`

Which are we going with? I would like consistency, regardless of which we choose.

@istoga @Sheyitofunmi @TaylorDJones11 @ibimina @Domicaren1 @Achele

**Review by msarit — APPROVED** [review]
Looking good! I assume you will import the pulled-out Search Form later, @istoga?

File: src/components/Header/NavBar.js (hunk 2 above)
Comment (ibimina) [reply src/components/Header/NavBar.js:1]: next/<item> is a component/element import from nextjs e.g Image, head, Link e.t.c
while @/path/to/item is a component import from a src directory

**Comment by istoga** [comment]
@msarit can you merge this please, I'm scared!

**Pull request merged**